src/Native/Runtime/inc/gcinfo.h

```diff
@@ -302,6 +302,49 @@ enum ScratchRegMask
     SR_MASK_LR = 0x40000,
 };
 
+#elif defined (_TARGET_WASM_)
```

Comment:
The changes in this file should not be needed anymore.

Reply:
Removed.